src/test/scala/wordbots/Randomizer.scala

```diff
@@ -0,0 +1,55 @@
+package wordbots
+
+import com.danielasfregola.randomdatagenerator.RandomDataGenerator
+import org.scalacheck.Gen.oneOf
+import org.scalacheck.rng.Seed
+import org.scalacheck.{Arbitrary, Gen}
+
+import scala.reflect.runtime.universe._
+import scala.util.{Random, Try}
+
+// NOTE: This takes a while to compile because of shapeless magic.
+object Randomizer extends RandomDataGenerator {
+  implicit lazy val arbInt: Arbitrary[Int] = Arbitrary(oneOf(0, 1, 2, 3))
+
+  import Semantics._
+
+  def action(): Option[Action] = safeRandom[Action]() match {
+    case Some(Instead(_)) => None
```

Comment:
What does `Some(Instead(_))` mean?

Reply:
Oh, I had a comment here before but I deleted it for some reason.

Basically, when I generate random actions, I want to ignore ones that start with `Instead(...)`, because that's a special kind of action that only occurs within triggered abilities (e.g. "When this creature deals damage, draw a card instead") and `CodeGenerator` doesn't handle it.